qiskit_device_benchmarking/bench_code/bell/partition.py

```diff
@@ -0,0 +1,204 @@
+# from qiskit-experiments-internal/qiskit_experiments_internal/tiled_experiment/partition.py
+"""
+Functions to partition nodes and edges into groups, such that nodes/edges
+in a group are distant from each other
+"""
+
+import rustworkx as rx
+
+
+def build_coupling_graph(backend, multigraph=False):
```

Comment:
this is here https://github.com/qiskit-community/qiskit-device-benchmarking/blob/main/qiskit_device_benchmarking/utilities/graph_utils.py#L95

Reply:
removed this module and using the graph utils block like in your bell state tomo notebook